modules/gltf/editor/editor_import_blend_runner.cpp

```diff
@@ -180,6 +180,10 @@ Error EditorImportBlendRunner::start_blender(const String &p_python_script, bool
 }
 
 Error EditorImportBlendRunner::do_import(const Dictionary &p_options) {
+	String blender_path = EDITOR_GET("filesystem/import/blender/blender_path");
+	ERR_FAIL_COND_V_MSG(blender_path.is_empty(), FAILED, "Blender path is empty, check your Editor Settings.");
+	ERR_FAIL_COND_V_MSG(!FileAccess::exists(blender_path), FAILED, vformat("Invalid Blender path: %s, check your Editor Settings.", blender_path));
```

Comment:
I think the validation should be done earlier, like at the start of `EditorSceneFormatImporterBlend::import_scene` (which calls this method).

Reply:
Excellent suggestion. At the same time, I saw a potential issue where the path was changed in the Editor Settings for a new version of blender (different path) but `blender_major_version` and `blender_minor_version` would never be updated until next editor restart.